### ISSUE TITLE
Change 127.0.0.1 by server ip

### ISSUE THREAD

**Issue by hugoforat** [daxeel/blockshell#8]
Hello,

How can i change "127.0.0.1:5000" by "mylocalip:5000" ?

Blockshell is on my linux server and I would like access to web page from my windows client.

thanks !

**Comment by sreecodeslayer** [comment]
For that, it is better to serve the Flask server behind Nginx/Apache using Gunicorn or uWSGI bound to `0.0.0.0:5000` and then use your Linux server's IP to access the web page.

If you are on the same network, you could add `host` and `port` to the [line](https://github.com/daxeel/blockshell/blob/fd43a22a6a3285eb5782a24a1c627428f48a5133/web.py#L45) as `app.run(host='0.0.0.0', port=5000, debug=True)`, and then access the page over `192.168.xx.xx` or what ever your internal IP is.

**Comment by smitthakkar96** [comment]
Maybe add a flag that can allow to specify port and host

**Comment by smitthakkar96** [comment]
@daxeel you never close issue with confirming from the issue creator unless inactive

**Comment by daxeel** [comment]
@smitthakkar96 Yes, you are absolutely correct. @sreecodeslayer explained about the issue thats why i thought that issue may be solved. By the way, I apologize for this action. Re-opening.
